subscription.go

```diff
@@ -308,8 +309,17 @@ func (s *Subscription) Unsubscribe() error {
 	return nil
 }
 
+// Close remove subscription from client's subs map
+func (s *Subscription) Close() {
+	s.status = SUBCLOSED
```

Comment:
Write should be protected by Lock, also if Subscription is already closed then s.centrifuge.removeSubscription should not be called again (i.e. when `sub.Close()` called two times).